docs/concepts/index.md

```diff
@@ -4,23 +4,21 @@ Remote Data Blocks allows you to integrate remote data into posts, pages, patter
 
 ## Remote data
 
-**Remote data** refers to data that is fetched from an external source, such as an API or a third-party service. This might be a product in your Shopify store, data in an Airtable or Google Sheet, or a file in a GitHub repository. Remote data is usually fetched via HTTP requests but you can [extend the plugin](extending.md) to support other transports.
+**Remote data** refers to data that is fetched from an external source, such as an API or a third-party service. This might be a product in your Shopify store, data in an Airtable or Google Sheet, or a file in a GitHub repository. Remote data is usually fetched via HTTP requests but you can [extend the plugin](../extending/index.md) to support other transports.
 
 ## Remote data block
 
 A **remote data block** is a custom block that fetches and displays data from a specific remote source. Each remote data block has a unique name and provides a specific kind of data.
 
-For example, you might have a remote data block named "Shopify Product" that fetches a product from your Shopify store and displays the product's name, description, price, and image. Or, you might have a remote data block named "Conference event" that displays rows from an Airtable and displays the event's name, location, and type. It might look something like this:
-
-[image tk]
```

Comment:
@chriszarate I removed this `[image tk]` placeholder (I think) and the prior sentence. I don't have any good visual examples on hand to provide an actual image, but I'd be happy to add this back if you have any good ideas.